### PR TITLE
net-news/rssguard: 3.3.5 version bump

### PR DESCRIPTION
**Pull request by a17r**
Fix HOMEPAGE, SRC_URI
Bump EAPI to 6
Unconditionally Qt5-based

Gentoo-bug: 571028

@gentoo/qt @kensington

### CODE REVIEW

File: net-news/rssguard/metadata.xml [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-    <remote-id type="bitbucket">skunkos/rssguard</remote-id>
-  </upstream>
+	<maintainer type="person">
+		<email>yngwin@gentoo.org</email>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">martinrotter/rssguard</remote-id>
+	</upstream>
+	<use>
+		<flag name="webengine">Use dev-qt/qtwebengine for internal web browser</flag>
```
Comment (Pesa) [inline net-news/rssguard/metadata.xml]: You can use a `<pkg>` tag here. Also maybe s/internal/embedded/

File: net-news/rssguard/rssguard-3.3.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+"
+DEPEND="${RDEPEND}
+	>=dev-qt/linguist-tools-5.6:5
+"
+
+src_configure() {
+	eqmake5 \
+		-r CONFIG+=$(usex debug debug release) \
+		USE_WEBENGINE=$(usex webengine true false) \
+		PREFIX=/usr \
```
Comment (Pesa) [inline net-news/rssguard/rssguard-3.3.5.ebuild]: `${EPREFIX}/usr`?

File: net-news/rssguard/rssguard-3.3.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	>=dev-qt/qtxmlpatterns-5.6:5
+	webengine? ( >=dev-qt/qtwebengine-5.6:5[widgets] )
+"
+DEPEND="${RDEPEND}
+	>=dev-qt/linguist-tools-5.6:5
+"
+
+src_configure() {
+	eqmake5 \
+		-r CONFIG+=$(usex debug debug release) \
```
Comment (Pesa) [inline net-news/rssguard/rssguard-3.3.5.ebuild]: is `-r` actually necessary? In any case, put the `CONFIG+=...` on a separate line.

Comment (a17r) [inline net-news/rssguard/rssguard-3.3.5.ebuild]: was part of the upstream readme, but doesn't actually seem to make a difference.

File: net-news/rssguard/rssguard-3.3.5.ebuild (hunk 2 above)
Comment (a17r) [inline net-news/rssguard/rssguard-3.3.5.ebuild]: done, thx!

File: net-news/rssguard/rssguard-3.3.5.ebuild [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+"
+DEPEND="${RDEPEND}
+	>=dev-qt/linguist-tools-5.6:5
+"
+
+src_configure() {
+	eqmake5 \
+		CONFIG+=$(usex debug debug release) \
+		USE_WEBENGINE=$(usex webengine true false) \
+		PREFIX=${EPREFIX}/usr \
```
Comment (Pesa) [inline net-news/rssguard/rssguard-3.3.5.ebuild]: quote EPREFIX (or the whole path)

**Comment by kensington** [comment]
Thanks, picked as 436fc595c838251bfa1dfece3cd572c352316f4d with some minor fixes squashed as discussed.

**Pull request closed without merging**